packages/server/routes/layout.tsx

```
@@ -2,11 +2,15 @@ interface LayoutInterface {
   children: React.ReactNode;
 }
 
-const Layout = ({ children }: LayoutInterface) => {
+const Layout = async ({ children }: LayoutInterface) => {
+  const test = await Promise.resolve("Hello World");
+
+  console.log(test);
+
   return (
     <html>
```

Biome (2.1.2)

[error] Provide a lang attribute when using the html element.

Setting a lang attribute on HTML document elements configures the languageused by screen readers when no user default is specified.

(lint/a11y/useHtmlLang)